packages/components/skeleton/src/SkeletonItem.styles.ts

```diff
@@ -0,0 +1,37 @@
+import { cva, VariantProps } from 'class-variance-authority'
+
+export const skeletonItemStyles = cva(['bg-neutral/dim-4', 'min-h-lg min-w-lg'], {
```

Comment:
`min-h-lg min-w-lg` -> `min-size-lg`

Reply:
Are you sure this class exists? I can't make it work :thinking: